.github/workflows/pre-merge.yaml

```diff
@@ -0,0 +1,53 @@
+---
+name: pre-merge
+
+on:
+  push:
+    branches:
+    - 'user/**'
+    - 'feature/**'
+    - 'improvement/**'
+    - 'bugfix/**'
+    - 'w/**'
+    - 'q/**'
+    - 'dependabot/**'
```

Comment:
are there branches we don't want to run on? this seems pretty inclusive.

Reply:
:thinking: only one I can think is `development/**`

Should we change it to something like this?
```yaml
  push:
    branches-ignore:
    - 'development/**'
```

Reply:
we could but I don't see much of an issue with running these tests on development as well, I think the confusing-ish part is that we call it the "pre-merge" pipeline when I think just calling it "tests" or "validations" or something similar to that effect. The fact that it happens in pre-merge is cool but irrelevant to what is actually happening. And we can run these pre, post or whenever. tl;dr name pipelines for what it does not when it happens

Reply:
You're right, I've renamed the pipeline